components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/ParentalAgeController.java

```diff
@@ -181,7 +184,11 @@ public void writeJSON(Patient patient, JSONObject json, Collection<String> selec
         for (String property : getProperties()) {
             if (data.has(property)) {
                 int age = data.getInt(property);
```

Comment:
we should always use optInt(property,0) to correctly handle misformatted data (will also handle empty strings ans nulls instead of 0s as indicators for the missing age)